src/databricks/labs/ucx/config.py

```diff
@@ -74,6 +74,9 @@ class WorkspaceConfig:  # pylint: disable=too-many-instance-attributes
     # [INTERNAL ONLY] Whether the assessment should lint only specific dashboards.
     include_dashboard_ids: list[str] | None = None
 
+    # [INTERNAL ONLY] Whether managed table should be created as external UC table.
+    managed_table_migration_option: int = 0
```

Comment:
```suggestion
    managed_tables_on_external_storage: str = 'CLONE'
```
1) be very explicit about this config property name
2) let's use uppercase strings as values
3) we have to document this property for public use, as there are thousands of folks already doing table migrations